tests/fast/TenantCycle.toml

```diff
@@ -0,0 +1,30 @@
+[configuration]
+allowDefaultTenant = false
+allowDisablingTenants = false
+
+[[test]]
+testTitle = 'TenantCreation'
+    [[test.workload]]
+    testName = 'CreateTenant'
+    name = 'First'
+
+    [[test.workload]]
+    testName = 'CreateTenant'
+    name = 'Second'
+
+[[test]]
+testTitle = 'Cycle'
+
+    [[test.workload]]
```

Comment:
For some odd reason, this particular test doesn't work even without tenants in main.

If I, in main, modify tests/fast/CycleTest.toml to include second Cycle workload (in the Unclogged test), and give it transactionsPerSecond = 2500.0 like here, I get an assertion error:
```
Assertion impl.has_value() failed @ /mnt/ephemeral/mdvorsky/foundationdb/flow/Arena.h 267:
std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >::__is_long() const at /usr/local/bin/../include/c++/v1/string:1456
 (inlined by) std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >::__get_pointer() const at /usr/local/bin/../include/c++/v1/string:1550
 (inlined by) std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >::data() const at /usr/local/bin/../include/c++/v1/string:1262
 (inlined by) std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >::c_str() const at /usr/local/bin/../include/c++/v1/string:1260
 (inlined by) internal_error_impl(char const*, char const*, int) at /mnt/ephemeral/mdvorsky/foundationdb/flow/Error.cpp:66
Optional<Standalone<StringRef> >::get() const & at /mnt/ephemeral/mdvorsky/foundationdb/flow/Arena.h:267
 (inlined by) CycleWorkload::CycleClientActorState<CycleWorkload::CycleClientActor>::a_body1loopBody1cont1loopBody1cont2(Optional<Standalone<StringRef> > const&, int) at /mnt/ephemeral/mdvorsky/foundationdb/fdbserver/workloads/Cycle.actor.cpp:119
...
```

It works when I set transactionsPerSecond to 250 in both copies of the workload. 

(A single Cycle workload with transactionsPerSecond = 2500 also fails, but in a different way).